fastlane.gemspec

```diff
@@ -103,8 +103,9 @@ Gem::Specification.new do |spec|
 
   # The Google API Client gem is *not* API stable between minor versions - hence the specific version locking here.
   # If you upgrade this gem, make sure to upgrade the users of it as well.
```

Comment:
Does this comment still apply?

Reply:
But there's no reason to keep them as is. Will revise it.

Reply:
I don't know whether the new gems are intended to use semantic versioning. I only noticed that these are no longer version locked.

Reply:
That's right! Thanks.

Reply:
Obviously these are different ecosystems, but if the versions line up at all, at least for javascript the google-apis follows semantic versioning, but it's essentially nothing but major versions because of it - there is always some API changing incompatibly: https://github.com/googleapis/google-api-nodejs-client/blob/master/CHANGELOG.md